hawkular-metrics/hawkular-metrics-wrapper.sh

```diff
@@ -18,58 +18,39 @@
 
 echo $(date "+%Y-%m-%d %H:%M:%S") Starting Hawkular Metrics
 
-# Set up the parameters to pass to EAP while removing the ones specific to the wrapper
-as_args=
-
-for args in "$@"
-do
-  if [[ $args == --hmw\.* ]]; then
-    case $args in
-      --hmw.keystore=*)
-        KEYSTORE="${args#*=}"
-        ;;
-      --hmw.truststore=*)
-        TRUSTSTORE="${args#*=}"
-        ;;
-      --hmw.keystore_password=*)
-        KEYSTORE_PASSWORD="${args#*=}"
-        ;;
-      --hmw.keystore_password_file=*)
-        KEYSTORE_PASSWORD_FILE="${args#*=}"
-        ;;
-      --hmw.truststore_password=*)
-        TRUSTSTORE_PASSWORD="${args#*=}"
-        ;;
-      --hmw.truststore_password_file=*)
-        TRUSTSTORE_PASSWORD_FILE="${args#*=}"
-        ;;
-      --hmw.jgroups_keystore=*)
-        JGROUPS_KEYSTORE="${args#*=}"
-        ;;
-      --hmw.jgroups_keystore_password_file=*)
-        JGROUPS_KEYSTORE_PASSWORD_FILE="${args#*=}"
-        ;;
-     --hmw.jgroups_keystore_password=*)
-        JGROUPS_KEYSTORE_PASSWORD="${args#*=}"
-        ;;
-      --hmw.jgroups_alias_file=*)
-        JGROUPS_ALIAS_FILE="${args#*=}"
-        ;;
-      --hmw.jgroups.alias=*)
-        JGROUPS_ALIAS="${args#*=}"
-        ;;
-    esac
-  else
-    as_args="$as_args $args"
-  fi
-done
+KUBERNETES_API_VERSION=${KUBERNETES_API_VERSION:-v1}
```

Comment:
See the comment I wrote on the Cassandra wrapper. But mostly, the changes here are simpler: for Wildfly, we build the keystore to contain the cert to use on the server's HTTPS configuration, and we add the service-ca to the trust store (so that it can trust the Cassandra's cert).